simple_history/tests/tests/test_models.py

```diff
@@ -1185,6 +1185,18 @@ def test_restore_pollwithexclude(self):
         original = historical.instance
         self.assertEqual(original.pub_date, poll.pub_date)
 
+    def test_get_default_values_for_excluded_fields(self):
```

Comment:
Can you add a test where the excluded field doesn't have a default? Also, unsure whether it the excluded fields should default to the default or just be empty, but I agree that it shouldn't error out

Reply:
Yes, i will update tests. As for the defaults, I agree with you that it would be better not to pass the excluded fields - let the Django itself fill them for instance. I will update PR soon.

Reply:
👍 thanks!

Reply:
Done